test/tests/materials/interface_material/tests

```diff
@@ -29,4 +29,12 @@
     recover = false
     method = '!dbg' # issue 12033
   []
+  [boundary_elem_aux]
+    type = Exodiff
+    input = interface_value_material_noIK.i
+    exodiff = interface_value_material_noIK_out.e
+    requirement = 'The system shall evaluate interface materials such that they can be used in boundary restricted elemental auxiliary kernels'
+    issues = '#14387'
+    design = 'Materials/index.md'
```

Comment:
you need to add 

```
    mesh_mode = 'REPLICATED'
    recover = false
    recover = false
    method = '!dbg' # issue 12033
    method = '!dbg' # issue 12033
```

Reply:
I hope not since this isn't using `BreakMeshByBlockGenerator`. This passed testing both in debug and with recover. The parallel failures should be fixed by 9ac82fe